### ISSUE TITLE
[CPPREST][C++]Accessors for boolean fields are not given a method name starting with "is"

### ISSUE THREAD

**Issue by javakrieg** [swagger-api/swagger-codegen#6854]
##### Description

If you have a boolean property in a definition file (e.g. "fixed"), the Java code generator will produce a setter called "setFixed" and a getter called "isFixed".  This is expected.  However, the CppRest/C++ code generator will produce a setter called "setFixed" but the getter will be called "getFixed".  This is incorrect.  It too should be "isFixed".

##### Swagger-codegen version

2.2.3

##### Swagger declaration file content or url

```yaml
  Parameter:
    type: object
    properties:
      name:
        type: string
      fixed:
        type: boolean
        description: True if the parameter cannot be changed, false otherwise.
```

**Comment by wing328** [comment]
@javakrieg that's fixed in the latest master (2.3.0). Please pull the latest or use the SNAPSHOT version mentioned in the README.

If it's still an issue, we will reopen this. Closing it for the time being.

**Comment by stkrwork** [comment]
its not in the abstract cpp codegen, but it can be quickly added. Here is the fix: #6880

**Comment by wing328** [comment]
@javakrieg the fix by @stkrwork has been merged into master. Please pull the latest master or use the SNAPSHOT version mentioned in the README to give it another try.
